suse_openstack_cloud

```diff
@@ -395,14 +395,14 @@ fi
 ###############################################################
 section_header "RabbitMQ broker"
 
-if [ -e /usr/sbin/rabbitmqctl ]; then
-    rabbitmq_tmpdir=$(mktemp -d)
-    rabbitmq_report_log=$rabbitmq_tmpdir/rabbitmq_report.log
+rabbitmqctl=/usr/lib64/rabbitmq/bin/rabbitmqctl
```

Comment:
Btw. I was thinking that we would like only to change directory for rabbitmqctl report here and discussion about main fix and adding rabbitmqctl support to supportconfig we had in https://github.com/SUSE-Cloud/supportutils-plugin-suse-openstack-cloud/pull/33